shared/chat/avatars.tsx

```diff
@@ -57,7 +57,7 @@ const MutedIcon = (p: {
 
 class Avatars extends React.Component<AvatarProps> {
   shouldComponentUpdate(nextProps: AvatarProps) {
-    return !shallowEqual(this.props, nextProps, (obj, oth, key) => {
+    return !shallowEqual(this.props, nextProps, (_, __, key) => {
```

Comment:
Is that second argument supposed to be a double underscore? Guessing it somehow distinguishes itself from the previous underscore?

Reply:
these are params, not object keys